### PR TITLE
update notif

### PR DESCRIPTION
**Pull request by forenZZy**

### CODE REVIEW

File: controllers/sendEmail.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+exports.sendEmail = (mailOptions) => {
+    var transporter = nodemailer.createTransport({
+        service: 'gmail',
+        auth: {
+            user: process.env.USER_EMAIL,
+            pass: process.env.USER_PASS,
+        }
+    });
+    // TODO remove console.log
+    transporter.sendMail(mailOptions, function (error, info) {
```
Comment (InnaKutsik) [inline controllers/sendEmail.js]: pls use promises or async/await

nodemailer support it

File: services/registrationNotification.js [hunk 2]
```diff
@@ -1,22 +1,12 @@
...
-        }
-    });
-
-    var mailOptions = {
+exports.sendNotif = (user) => {
+    const mailOptions = {
         from: '"Sport News" <process.env.USER_EMAIL>',
         to: user,
         subject: 'Subscription in Sport News',
         html: `<table style="background:#f9f9fb;width: 100%;padding: 0 20%;">
```
Comment (InnaKutsik) [inline services/registrationNotification.js]: pls move template to some template directory with registrationNotificationTemplate name

File: templates/registrationNotificationTemplate.js [hunk 3]
```diff
@@ -1,22 +1,12 @@
...
+exports.sendRegNotification = (user) => {
+    const mailOptions = {
         from: '"Sport News" <process.env.USER_EMAIL>',
         to: user,
         subject: 'Subscription in Sport News',
         html: `<table style="background:#f9f9fb;width: 100%;padding: 0 20%;">
-                <thead>
-                    
+                <thead>   
                 </thead>
```
Comment (anna-kuzii) [inline templates/registrationNotificationTemplate.js]: do we need this empty tag?

File: server.js [hunk 4]
```diff
@@ -11,6 +9,8 @@ app.get('/', (req, res) => {
 
 const PORT = process.env.PORT || 5000;
 
+
+
```
Comment (anna-kuzii) [inline server.js]: why?

Comment (forenZZy) [reply server.js]: deleted

File: templates/registrationNotificationTemplate.js (hunk 3 above)
Comment (forenZZy) [reply templates/registrationNotificationTemplate.js]: deleted

**Pull request merged**